pkg/mountutil/mountutil.go

```diff
@@ -90,7 +93,7 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 		return nil, errors.Errorf("failed to parse %q", s)
 	}
 	res.Mount = specs.Mount{
-		Type:        "none",
+		Type:        vType,
```

Comment:
Again, you can't have "volume" "bind" here

https://github.com/opencontainers/runtime-spec/blob/master/config.md#posix-platform-mounts

Reply:
The type fIeld can be  in this struct https://github.com/containerd/nerdctl/blob/86e8f5f10d46fc51fb152733a025d940f91ae668/pkg/mountutil/mountutil.go#L32

Reply:
@AkihiroSuda moved.